controllers/hcpvaultsecretsapp_controller.go

```diff
@@ -28,8 +30,16 @@ import (
 	"github.com/hashicorp/vault-secrets-operator/internal/credentials"
 	"github.com/hashicorp/vault-secrets-operator/internal/credentials/hcp"
 	"github.com/hashicorp/vault-secrets-operator/internal/helpers"
+	"github.com/hashicorp/vault-secrets-operator/internal/version"
 )
 
+const (
+	headerHCPCLIRequester = "X-HCP-CLI-Requester"
```

Comment:
```suggestion
	headerHCPCLIRequester = "x-hvs-requester"
```

Reply:
Good catch! Fixed in 3342a4ad3a0fdb0f0dff6ababeb6eb5cc3c4541f . Since HTTP headers are case-insensitive I've preserved the capitalization.